example/src/shared/Wysiwyg/index.js

```diff
@@ -5,7 +5,7 @@ import WysiwygFooter from './WysiwygFooter';
 import WysiwygExpand from './WysiwygExpand';
 import MediaLibrary from './MediaLibrary';
 import Editor from './Editor';
-import { ButtonText } from '@strapi/parts/Text';
+import { TextButton } from '@strapi/parts/Text';
```

Comment:
The component that is exported from Text is ButtonText, not TextButton :). If it does not work, make sure to yarn setup at the project run, and reinstall the example dependencies. 😊